v2/googlecloud-to-splunk/docs/GCSToSplunk/gcs-to-splunk-image-spec.json

```diff
@@ -0,0 +1,128 @@
+{
+  "defaultEnvironment": {},
+  "image": "gcr.io/project-id/image-name",
+  "metadata": {
+    "description": "A pipeline that reads from a GCS bucket and writes to Splunk's HTTP Event Collector (HEC).",
+    "name": "GCS to Splunk",
+    "parameters": [
+      {
+        "helpText": "GCS files to read the input. Example: gs://mybucket/test-*.csv",
+        "isOptional": false,
+        "label": "GCS file pattern to search for CSV files",
+        "name": "inputFileSpec",
+        "paramType": "TEXT",
+        "regexes": [
+          "^gs:\\/\\/[^\\n\\r]+$"
+        ]
+      },
+      {
+        "helpText": "True if CSVs include headers or False if they do not.",
```

Comment:
The help text implies "True" or "False" are values, but the regex wouldn't allow those. Please update this to valid values and mention the default. It should also make a note that all the files from `inputFileSpec` must hold to this option.

Note that everything here is what would appear in the UI if someone is going through the Cloud Console. It's arguably more important to be helpful here than in the README.